webknossos/webknossos/dataset/dataset.py

```diff
@@ -396,6 +396,56 @@ def open(cls, dataset_path: Union[str, PathLike]) -> "Dataset":
             exist_ok=True,
         )
 
+    @classmethod
+    def announce_manual_upload(
+        cls,
+        dataset_name: str,
+        organization: str,
+        initial_team_ids: List[str],
+        folder_id: str,
+        token: Optional[str] = None,
+    ) -> None:
+        """
+        Announces a manual dataset upload to webknossos. This is useful when users with access
+        to the file system of the datastore want to upload a dataset manually. It creates an entry
+        in the database and sets the access rights accordingly.
+        """
+        from ..client._upload_dataset import _cached_get_upload_datastore
+        from ..client.api_client.models import ApiDatasetAnnounceUpload
+        from ..client.context import _get_context
+
+        context = _get_context()
+        dataset_announce = ApiDatasetAnnounceUpload(
+            dataset_name=dataset_name,
+            organization=organization,
+            initial_team_ids=initial_team_ids,
+            folder_id=folder_id,
+        )
+        token = token or context.token
+        upload_url = _cached_get_upload_datastore(context)
+        datastore_api = context.get_datastore_api_client(upload_url)
+        datastore_api.dataset_reserve_manual_upload(dataset_announce, token=token)
+
+    @classmethod
+    def trigger_reload(
```

Comment:
Do all datasets have this method? I’m not sure how to express that this is only for datasets present in a webknossos datastore :thinking: If you only create the dataset locally, this does not make sense. Maybe you have a good idea

Reply:
I don't have a good idea either. We could rename the method to `trigger_reload_in_datastore`, make the docstring more specific or put it somewhere else in the libs, like a new datastore class or a collection of wk-api utils, do you have any preferences?

Reply:
I guess with renaming + better docstring we are already on a good way :)